bigtable-hbase-dataflow/src/main/java/com/google/cloud/bigtable/dataflow/CloudBigtableConfiguration.java

```diff
@@ -184,20 +185,21 @@ public Configuration toHBaseConfig() {
   }
 
   /**
-   * Creates a new Builder from the configuration
-   * @return A new {@link Builder}
+   * Creates a new {@link Builder} object from the configuration.
+   * @return A new {@link Builder}.
```

Comment:
I think we should have something that informs the user that this isn't a new Builder, but rather a Builder that contains the configuration that this CloudBigtableConfiguration contains.